### ISSUE TITLE
helm with tramp stopped working on uppmax

### ISSUE THREAD

**Issue by percyfal** [percyfal/emacs.d#1]
I had to change default tramp method to sftp. However problems persist when using helm as the buffer stalls. Would be nice to disable helm for tramp only. See 
https://github.com/emacs-helm/helm/issues/749

for related issue

**Comment by percyfal** [comment]
Test recent version of tramp; see

http://stackoverflow.com/questions/33442144/how-do-i-log-history-in-a-tramp-emacs-ssh-session

**Comment by thokall** [comment]
Hi,

Is this issue really a helm one? Does your tramp connection to Uppmax work these days? My ability to connect via tramp stopped working around new year using v. 24, but would still work using v. 22. I have not tried with a new version of tramp, but using ksh instead of sh or bash does work. I am however all ears for a better solution.

Thomas, BILS

**Comment by percyfal** [comment]
No, probably not helm, but I have used tramp with emacs 24 and 25 without problems until Dec 10 2015. I have no clue what's going on.

**Comment by thokall** [comment]
I have no idea either but the command that my tramp connection sends when it connects to uppmax 
```exec env ENV='' HISTFILE=/dev/null PROMPT_COMMAND='' PS1=\#\$\  PS2='' PS3='' /bin/sh```
leads to a segmentation fault due to the HISTFILE setting. The same goes for /bin/bash, but with /bin/ksh it works again. Not sure how one should alter this in the correct way so currently I am just altering the tramp-methods in my .emacs file
